editor/editor_inspector.cpp

```diff
@@ -1315,12 +1319,14 @@ void EditorInspectorSection::_notification(int p_what) {
 			if (dropping) {
 				dropping_unfold_timer->start();
 			}
+			call_deferred(SNAME("update"));
```

Comment:
Does it need to be deferred? If there's indeed some specific reason then adding a comment about it might be a good idea.
(same for the other `call_deferred()`)

Reply:
I just tested it, and it seems like it didn't redraw in my earlier testing because of some other reason.